src/vs/platform/terminal/common/xterm/shellIntegrationAddon.ts

```diff
@@ -224,6 +225,21 @@ const enum VSCodeOscPt {
 	 * WARNING: This sequence is unfinalized, DO NOT use this in your shell integration script.
 	 */
 	SetMark = 'SetMark',
+
+	/**
+	 * Sends the shell's environment.
+	 *
+	 * Format: `OSC 633 ; Env ; <Environment> ; <Nonce>`
+	 *
+	 *	`Environment` - A JSON string containing the shell's environment variables.
+	 *   TODO: Encoding information for commandline sequence.
```

Comment:
Mention encoding information like in `CommandLine`

Reply:
@Tyriar Do you think https://github.com/microsoft/vscode/pull/237415/commits/6b081da090c76e37eb60e9a09cdc2f474196bea3 is enough? 

Since we are using the same escaping [function](https://github.com/microsoft/vscode/blob/b0d6d34fbbdd73e273715a6f68e5dfbc7bf6147b/src/vs/workbench/contrib/terminal/common/scripts/shellIntegration.ps1#L121 ) as commandLine, I included a short single line.

Reply:
Yes, but let's add a link that will work in hovers via `{@link CommandLine}`